### PR TITLE
Add notes to articles

### PR DESCRIPTION
**Pull request by kwalrath**
Contributes to #663.

Staged:
https://kw-www-dartlang-1.firebaseapp.com/articles/dart-vm/simd
https://kw-www-dartlang-1.firebaseapp.com/articles/dart-vm/numeric-computation
https://kw-www-dartlang-1.firebaseapp.com/articles/dart-vm/benchmarking
https://kw-www-dartlang-1.firebaseapp.com/articles/libraries/serialization
https://kw-www-dartlang-1.firebaseapp.com/articles
https://kw-www-dartlang-1.firebaseapp.com/articles/dart-vm

### CODE REVIEW

File: src/_articles/dart-vm/benchmarking.md [hunk 1]
```diff
@@ -2,23 +2,22 @@
...
-nor does it reflect the changes that are coming in [Dart 2](/dart-2).
+October 2012 (updated November 2012; note updated July 2018)_
+
+<aside class="alert alert-warning" markdown="1">
+  This article was initially targeted for the standalone VM under the
+  1.x Dart SDK.  It does not reflect the Flutter VM,
+  nor does it reflect changes in [Dart 2](/dart-2).
+  For example, [Dartium doesn't work with Dart 2.]({{site.webdev}}/dart-2#tools)
+  Also, checked mode and production mode no longer exist;
+  instead, tools have platform-specific support for expensive development-time
```
Comment (rmacnak-google) [inline src/_articles/dart-vm/benchmarking.md]: The assert mechanism is not platform-specific. It just happens that the Flutter framework includes a lot of expensive asserts.

File: src/_articles/dart-vm/numeric-computation.md [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
 category: dart-vm
 obsolete: true
 ---
 
 _Written by John Mccutchan <br>
-May 2013_
+May 2013 (note added July 2018)_
+
+<aside class="alert alert-warning" markdown="1">
+  In Dart 2, "proper" integers were replaced with 64-bit signed, wrapping
```
Comment (rmacnak-google) [inline src/_articles/dart-vm/numeric-computation.md]: "Proper integers" is my perspective. A better term for a general audience is "arbitrary-precision integers".

File: src/_articles/dart-vm/simd.md [hunk 3]
```diff
@@ -1,15 +1,21 @@
...
 
 _Written by John McCutchan <br>
-August 2013_
+August 2013 (note added July 2018)_
 
-Programs written in Dart can use new numeric types
+<aside class="alert alert-info" markdown="1">
+  This article's content is still relevant in Dart 2.
+  However, the examples haven't been tested recently,
+  and they use the now-optional `new` keyword.
```
Comment (rmacnak-google) [inline src/_articles/dart-vm/simd.md]: Not sure why "new" is relevant. It hasn't been removed from Dart 2.

File: src/_articles/dart-vm/numeric-computation.md (hunk 2 above)
Comment (kwalrath) [reply src/_articles/dart-vm/numeric-computation.md]: Thanks, that's better. :)

File: src/_articles/dart-vm/benchmarking.md (hunk 1 above)
Comment (kwalrath) [reply src/_articles/dart-vm/benchmarking.md]: I meant to say that it was platform specific how you turn on asserts (and any other platform-specific dev-time stuff). I'll try to clarify.

File: src/_articles/dart-vm/simd.md (hunk 3 above)
Comment (kwalrath) [reply src/_articles/dart-vm/simd.md]: It hasn't been removed, but the style guide says [DON'T use `new`](https://www.dartlang.org/guides/language/effective-dart/usage#dont-use-new), and we've removed `new` from all of our auto-included code.

**Comment by kwalrath** [comment]
PTAL @rmacnak-google. I've updated the staging server.

**Comment by kwalrath** [comment]
Thanks, @rmacnak-google!

**Pull request merged**